### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by smartm0use**
Fixed Rust version requirement

### CODE REVIEW

**Comment by RCasatta** [comment]
Need to investigate, CI is apparently testing for 1.65

**Comment by RCasatta** [comment]
from the docs this PR is right https://doc.rust-lang.org/stable/std/collections/struct.BTreeSet.html#method.first

File: Readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Locally can run on regtest.
 
 ## Running locally
 
-Supposing to have [rust installed](https://www.rust-lang.org/tools/install) >= 1.65.0 and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
+Supposing to have [rust installed](https://www.rust-lang.org/tools/install) > 1.65.0 and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
```
Comment (RCasatta) [inline Readme.md:13]: could you please use "1.66.0": (either "at least 1.66.0" or ">= 1.66.0") I think it's more clear

**Review by RCasatta — COMMENTED** [review]
Concept ACK (but I don't know exactly why CI is not catching this)
Please update also the github actions file to use 1.66.0

File: Readme.md (hunk 1 above)
Comment (smartm0use) [reply Readme.md]: Done.

**Comment by RCasatta** [comment]
Could you please:
1) Squash the two commits 7d52b18040a3afbe1854f9380b7efe9b8c985ec3 and 7d52b18040a3afbe1854f9380b7efe9b8c985ec3 you usually don't want different commits of the same PR to touch the same lines
2) in another or same commit, update here https://github.com/RCasatta/fbbe/blob/55a80c61ad8e30964eb2db5b4f3b7d3b1f0c9b01/.github/workflows/test.yml#L41 from 1.65 to 1.66

Thanks, sorry for being pedantic

**Comment by RCasatta** [comment]
utACK 03e3b19856efc691be7070c5308841862d096f7f

Thanks!

**Pull request merged**